public/css/_inputs.scss

```diff
@@ -131,8 +131,8 @@ label.checkbox {
 
   >ul {
     display: none;
-    position: absolute;
-    width: 100%;
+    position: fixed;
+    // width: 100%;
```

Comment:
recommend /*width: 100%;*/ instead as css doesn't understand // for single line comment

Reply:
I removed the width. No need to have the comment hang around.